testsuite/features/srv_content_lifecycle.feature

```diff
@@ -2,7 +2,8 @@
 # Licensed under the terms of the MIT license.
 
 Feature: Content lifecycle
-
+  
+  @SCC_CREDS
```

Comment:
Remove the two spaces added on previous line

`@scc_credentials` in column 1

Why do we need to make this conditional in this feature?